nifpga/session.py

```diff
@@ -6,9 +6,11 @@
 
 from .nifpga import (_SessionType, _IrqContextType, _NiFpga, DataType,
                      OPEN_ATTRIBUTE_NO_RUN, RUN_ATTRIBUTE_WAIT_UNTIL_DONE,
-                     CLOSE_ATTRIBUTE_NO_RESET_IF_LAST_SESSION)
+                     CLOSE_ATTRIBUTE_NO_RESET_IF_LAST_SESSION, FifoProperty,
+                     _fifo_properties_to_types, FlowControl, DmaBufferType,
+                     FpgaViState)
 from .bitfile import Bitfile
-from .status import InvalidSessionError
+from .status import IrqTimeoutWarning, InvalidSessionError
```

Comment:
It looks like a few things got merged incorrectly, like the IrqTimeoutWarning here.